ts/components/session/ActionsPanel.tsx

```diff
@@ -59,18 +59,33 @@ export class ActionsPanel extends React.Component<Props, State> {
           'refreshAvatarCallback'
         );
         setTimeout(async () => {
+          const disabledMultiDeviceCountDb = await window.Signal.Data.getItemById(
```

Comment:
We can call `getItemById` directly from tx

Reply:
fixed